scripts/emerge/stage_prs_anvil_deliverable_data.py

```diff
@@ -0,0 +1,193 @@
+    anvil_samples_df = prs_entities_df[["crsp_sample_id", "collaborator_sample_id", "collaborator_participant_id"]]
+    # rename entity column and make entity:entity_id column first in tsv
+    anvil_samples_df = anvil_samples_df.rename(columns={"crsp_sample_id": "entity:Samples_id"})
+    first_column = anvil_samples_df.pop("entity:Samples_id")
+    anvil_samples_df.insert(0, "entity:Samples_id", first_column)
+
+    # Arrays
+    # create arrays subset df
+    anvil_arrays_df = prs_entities_df[["red_idat_cloud_path", "green_idat_cloud_path",
+                                       "gtc_file", "arrays_variant_calling_detail_metrics_file",
+                                       "single_sample_vcf", "single_sample_vcf_index",
+                                       "imputed_single_sample_vcf", "imputed_single_sample_vcf_index",
+                                       "crsp_sample_id"]]
+    # rename entity column and make entity:entity_id column first in tsv
+    anvil_arrays_df = anvil_arrays_df.rename(columns={"crsp_sample_id": "entity:Arrays_id"})
+    first_column = anvil_arrays_df.pop("entity:Arrays_id")
+    anvil_arrays_df.insert(0, "entity:Arrays_id", first_column)
+
+    # Polygenic Risk Scores
+    # create prs_scores subset df
+    anvil_prs_scores_df = prs_entities_df[["ast_raw", "ast_adjusted", "ast_percentile", "brca_raw", "brca_adjusted",
+                                           "brca_percentile", "afib_raw", "afib_adjusted", "afib_percentile",
+                                           "chd_raw", "chd_adjusted", "chd_percentile", "ckd_raw", "ckd_adjusted",
+                                           "ckd_percentile", "hcl_raw", "hcl_adjusted", "hcl_percentile", "bmi_raw",
+                                           "bmi_adjusted", "bmi_percentile", "prca_raw", "prca_adjusted", "prca_percentile",
+                                           "t1d_raw", "t1d_adjusted", "t1d_percentile", "t2d_raw", "t2d_adjusted", "t2d_percentile",
+                                           "crsp_sample_id"]]
+    # rename entity column and make entity:entity_id column first in tsv
+    anvil_prs_scores_df = anvil_prs_scores_df.rename(columns={"crsp_sample_id": "entity:Polygenic_Risk_Scores_id"})
+    first_column = anvil_prs_scores_df.pop("entity:Polygenic_Risk_Scores_id")
+    anvil_prs_scores_df.insert(0, "entity:Polygenic_Risk_Scores_id", first_column)
+
+    # write individual dataframes to tsv files
+    anvil_arrays_df.to_csv("arrays.tsv", sep = "\t", index=False)
+    anvil_samples_df.to_csv("samples.tsv", sep="\t", index=False)
+    anvil_prs_scores_df.to_csv("prs_scores.tsv", sep="\t", index=False)
+
+    # load tsv files to destination workspace
+    for tsv in ["arrays.tsv", "samples.tsv", "prs_scores.tsv"]:
+        print(f"Loading {tsv} to {dest_workspace}.")
+        upload_data_table(tsv, dest_workspace, dest_namespace)
+
+
+def copy_object_to_bucket(src_bucket_name, src_object_name, dest_bucket_name, dest_object_name):
```

Comment:
minor: this function name is a little confusing since it specifies that the destination is a bucket but doesn't specify that the source is a bucket. fn name could be more intuitive, like `copy_object_between_buckets` or just `copy_object`